bin/rector.php

```diff
@@ -39,7 +38,7 @@ final class AutoloadIncluder
     public function includeDependencyOrRepositoryVendorAutoloadIfExists(): void
     {
         // Rector's vendor is already loaded
-        if (class_exists(RectorKernel::class)) {
+        if (class_exists(\Rector\Core\DependencyInjection\LazyContainerFactory::class)) {
```

Comment:
on running test, I still got infinite loop `segmentation fault` locally on some tests, eg:

```
vendor/bin/phpunit packages-tests/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser/ArrayParserTest.php

PHPUnit 10.3.1 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.2.5
Configuration: /Users/samsonasik/www/rector-src/phpunit.xml

[1]    9179 segmentation fault  vendor/bin/phpunit 
```

I guess there is flip flop bug on Laravel container on `$lazyRectorConfig->afterResolving()` usage?

Reply:
The `afterResolving` could be case. Could you share some miminal reproducer?

I'll see what i can do with my limited knowlege

Reply:
Oh, I just removed the `vendor`, and re-run composer update, it seems working ok on main branch now